### PR TITLE
Add v1.0.0 of mattermost-plugin-github to the Marketplace

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Add v1.0.0 of mattermost-plugin-github to the Marketplace.
Screenshot from 2020-05-28 21-00-14

#### Ticket Link
https://github.com/mattermost/mattermost-plugin-github/pull/274

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1,4 +1,111 @@
...
+    "download_url": "https://github.com/mattermost/mattermost-plugin-github/releases/download/v1.0.0/github-1.0.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-github/releases/tag/v1.0.0",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl7QCTIACgkQ0bVLR6XO/sS5lBAAkRyZ/fR6i2gNgynY+5YSf0h+SnzTqsmS/9YnQARVsu6PvoHLx+JdpeI8zNHtk3WsKuo2lERy7wk0KmQgGGZdFVagraVZU8gB/8LGAqyBUOOkr1mhBssjdJWoaYz7BNh6UjLrawBQcUqHKuw9L9/TrGuxvZ20If/zP5C4wX9Oe7dKeXQ+6qhelb2WM9z6U7TJQmGAd2xewZHBi4naESnZI5HxOtnwqO9uqZMooXLkk7Xn0EmIZ5dq572YoV9Noe7UF2GBSwaekskI/IjT1qGs5OjGtPX9tHMMQAEtJ83muNd66TAmqSmGT6olTe0+prRyo2V8KBOc1kw6WnitFBkjMTVEEmHFBtGXQGKMWludid9a6ljIT/jfdH+ZWGgFL1JSRoV7++/iDwQO3z7HW7jfrnwiSqdP575tvt9jdgkNQHWkBdXupGFjErWpuFaPR0frpGe/BzqwbjdiWUkBWyR/MDY1XbA43X4gyh8RSa4MHzPCUYfXoX3wJ6Y1HvT6x5si57e3GIooAH9Ebj3OE1Y/bEDJ419J5/yLwe7q0KGSQhYMd2spAgIgH/+PAbJ5W9uam4Gkb3PJ5CGr6n68WBWujLWoh5SCV0jipoiU2GvyfV/kJytl9JKfC82snslSoOfK76fh71pg2PGIy8X0460b46ElxfOjRj9vnvIjP+QeptE=",
+    "manifest": {
+      "id": "github",
+      "name": "GitHub",
+      "description": "GitHub plugin for Mattermost.",
```
Comment (aaronrothschild) [inline plugins.json:11]:
```suggestion
      "description": "GitHub plugin for Mattermost",
```

Comment (hanzei) [reply plugins.json:11]: @aaronrothschild Actually, most of the plugin descriptions end with a dot. Should this be holistically changed?
Screenshot from 2020-05-28 21-09-13

**Comment by DHaussermann** [comment]
Testing done.
- Version number is correct
- Fresh install successful  
- Upgrade successful 
- Icon looks good (including dark theme)

@hanzei one question on this. During the upgrade path, the user is cautioned that the new version may contain breaking changes and to review the release notes. 
I have reviewed the notes and there is no mention of a breaking change (as I expected). Is this intentional because we are moving from v0._x_ to v1.0.0?
Screen Shot 2020-05-30 at 3 46 35 PM

**Comment by hanzei** [comment]
@DHaussermann That is a great observation. I'm not entirely sure what the expected behaviour, but the current behavior doesn't seam wrong to me. The bump to v1.0.0 could could contain breaking changes. Let's discuss this on Mattermost.

**Pull request merged**